am-core-web-service/src/main/java/org/am/core/web/service/admingeneral/AcademicPeriodService.java

```diff
@@ -0,0 +1,113 @@
+package org.am.core.web.service.admingeneral;
+
+import org.am.core.web.domain.entity.admingeneral.AcademicPeriod;
+import org.am.core.web.dto.admingeneral.AcademicPeriodDto;
+import org.am.core.web.dto.admingeneral.AcademicPeriodRequest;
+import org.am.core.web.repository.jpa.CustomMap;
+import org.am.core.web.repository.jpa.admingeneral.AcademicPeriodRepository;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class AcademicPeriodService implements CustomMap<AcademicPeriodDto, AcademicPeriod> {
+    private final AcademicPeriodRepository academicPeriodRepository;
+
+    public AcademicPeriodService(AcademicPeriodRepository academicPeriodRepository) {
+        this.academicPeriodRepository = academicPeriodRepository;
+    }
+
+    @SuppressWarnings("UnresolvedClassReferenceRepair")
+    public List<AcademicPeriodDto> getActiveAcademicPeriodByAreaId(Integer areaID) {
```

Comment:
getActiveAcademicPeriodsByAreaId plural instead of singular because wa are getting a list